src/error.rs

```diff
@@ -1,3 +1,10 @@
+#[cfg(windows)]
+impl From<::std::io::Error> for Error {
```

Comment:
Wouldn't it be better to remove/extend the conditional from the 
`foreign_links` section below?

Reply:
done